packages/core/tests/base.test.ts

```diff
@@ -115,7 +115,14 @@ describe("Abby", () => {
         config1: "String",
         config2: "String",
       },
+      flags: ["flag1", "flag2"],
       settings: {
+        flags: {
+          fallbackValues: {
```

Comment:
for which ENV is this? all? we would need to have it for each ENV independently :)